schema/postgres/visibility/schema.sql

```diff
@@ -10,12 +10,16 @@ CREATE TABLE executions_visibility (
   history_length       BIGINT,
   memo                 BYTEA,
   encoding             VARCHAR(64) NOT NULL,
-  task_queue            VARCHAR(255) DEFAULT '' NOT NULL,
+  task_queue           VARCHAR(255) DEFAULT '' NOT NULL,
 
   PRIMARY KEY  (namespace_id, run_id)
 );
 
 CREATE INDEX by_type_start_time ON executions_visibility (namespace_id, workflow_type_name, status, start_time DESC, run_id);
 CREATE INDEX by_workflow_id_start_time ON executions_visibility (namespace_id, workflow_id, status, start_time DESC, run_id);
-CREATE INDEX by_status_by_close_time ON executions_visibility (namespace_id, status, start_time DESC, run_id);
+CREATE INDEX by_status_by_start_time ON executions_visibility (namespace_id, status, start_time DESC, run_id);
```

Comment:
@shawnhathaway - technically we can create a partial index here (https://www.postgresql.org/docs/8.0/indexes-partial.html). MySQL doesn't support this, but Postgres does. Let me know if you think it is worth doing the differentiation here.

Reply:
Imho, this should be fine as is unless it's very low hanging fruit effort-wise to add the partials. It adds size to index, but from my understanding high-performant visibility users will use Elastisearch. Let's sync today.